### PR TITLE
Done view detail-book

### PR DESCRIPTION
**Pull request by hungsondo**
## Related Tickets:
- [#41122](https://edu-redmine.sun-asterisk.vn/issues/41122)

## WHAT this PR does?
- Done template book detail for user
- Complete add, edit, delete review 
- Complete add. edit, delete comment

## Notes Impacted Areas:
- N/A.

## Attachments:
https://user-images.githubusercontent.com/44666485/132191736-927f5839-5424-4142-949f-7e55f05a61eb.mp4

### CODE REVIEW

**Review by longnv-99 — APPROVED** [review]

**Review by truongcr7 — APPROVED** [review]

**Comment by hungsondo** [comment]
ready

File: resources/views/book-detail.blade.php [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+                    <label class="form-check-label" for="inlineRadio3">
+                        <span>&#11088;</span><span>&#11088;</span><span>&#11088;</span>
+                    </label>
+                </div>
+                <div class="form-check form-check-inline">
+                    <input class="form-check-input" type="radio" name="rate" id="inlineRadio3" value={{ config('app.four-star') }}>
+                    <label class="form-check-label" for="inlineRadio3">
+                        <span>&#11088;</span><span>&#11088;</span><span>&#11088;</span><span>&#11088;</span>
+                    </label>
+                </div>
```
Comment (manhtd-1083) [inline resources/views/book-detail.blade.php:56]: mình suggest dùng for từ min rating tới max rating nhé

**Comment by hungsondo** [comment]
ready

**Comment by hungsondo** [comment]
ready

**Review by manhtd-1083 — APPROVED** [review]

**Pull request merged**